src/NLU.DevOps.ModelPerformance/TestCaseSource.cs

```diff
@@ -195,11 +174,8 @@ bool isNoneIntent(string intent)
                     new[] { expected, text },
                     "Utterances have matching intent.",
                     "Intent");
-
-                yield break;
             }
-
-            if (!isNoneIntent(expected))
+            else
             {
                 yield return FalseNegative(
```

Comment:
The point I was making about binary results was that we no longer need the concept of Positive or Negative, just true and false.